ArduCopter/mode_guided.cpp

```diff
@@ -6,10 +6,10 @@
  * Init and run calls for guided flight mode
  */
 
-static Vector3p guided_pos_target_neu_m;        // position target (used by posvel controller only)
-static bool guided_is_terrain_alt;              // true if guided_pos_target_neu_m.z is an alt above terrain
-static Vector3f guided_vel_target_neu_ms;       // velocity target (used by pos_vel_accel controller and vel_accel controller)
-static Vector3f guided_accel_target_neu_mss;    // acceleration target (used by pos_vel_accel controller vel_accel controller and accel controller)
+static Vector3p guided_pos_target_ned_m;        // position target (used by posvel controller only)
+static bool guided_is_terrain_alt;              // true if guided_pos_target_ned_m.z is an alt above terrain
```

Comment:
I wonder if this comment is still correct.  Is it really, "alt above terrain"?

Reply:
// true if guided_pos_target_ned_m.z should be offset by the terrain altitude